spec/lib/pdf_filler/f13614c_pdf_spec.rb

```diff
@@ -516,29 +516,131 @@
-            "form1[0].page1[0].yourLastName[0]" => "Heifer",
-            "form1[0].page1[0].yourTelephoneNumber[0]" => "(415) 816-1286",
-            "form1[0].page1[0].yourDateOfBirth[0]" => "4/19/1961",
-            "form1[0].page1[0].yourJobTitle[0]" => nil,
+                                                              # You
+                                                              "form1[0].page1[0].yourFirstName[0]" => "Hoofie",
+                                                              "form1[0].page1[0].yourMiddleInitial[0]" => nil,
+                                                              "form1[0].page1[0].yourLastName[0]" => "Heifer",
+                                                              "form1[0].page1[0].yourTelephoneNumber[0]" => "(415) 816-1286",
+                                                              "form1[0].page1[0].yourDateOfBirth[0]" => "4/19/1961",
+                                                              "form1[0].page1[0].yourJobTitle[0]" => nil,
 
-            # Spouse
-            "form1[0].page1[0].youSpouseWereIn[0].column2[0].totallyPermanentlyDisabled[0].disabledYou[0]" => "1",
-            "form1[0].page1[0].youSpouseWereIn[0].column2[0].legallyBlind[0].legallyBlindNo[0]" => "1",
-            "form1[0].page1[0].spousesFirstName[0]" => 'Hattie',
-            "form1[0].page1[0].spousesMiddleInitial[0]" => nil,
-            "form1[0].page1[0].spousesLastName[0]" => 'Heifer',
-            "form1[0].page1[0].spousesTelephoneNumber[0]" => nil,
-            "form1[0].page1[0].spousesDateOfBirth[0]" => "11/1/1959",
-            "form1[0].page1[0].spousesJobTitle[0]" => nil,
-            "form1[0].page1[0].youSpouseWereIn[0].column2[0].totallyPermanentlyDisabled[0].disabledYou[0]" => "1",
-            "form1[0].page1[0].youSpouseWereIn[0].column2[0].legallyBlind[0].legallyBlindNo[0]" => "1",
-          })
+                                                              # Spouse
+                                                              "form1[0].page1[0].youSpouseWereIn[0].column2[0].totallyPermanentlyDisabled[0].disabledYou[0]" => "1",
+                                                              "form1[0].page1[0].youSpouseWereIn[0].column2[0].legallyBlind[0].legallyBlindNo[0]" => "1",
+                                                              "form1[0].page1[0].spousesFirstName[0]" => 'Hattie',
+                                                              "form1[0].page1[0].spousesMiddleInitial[0]" => nil,
+                                                              "form1[0].page1[0].spousesLastName[0]" => 'Heifer',
+                                                              "form1[0].page1[0].spousesTelephoneNumber[0]" => nil,
+                                                              "form1[0].page1[0].spousesDateOfBirth[0]" => "11/1/1959",
+                                                              "form1[0].page1[0].spousesJobTitle[0]" => nil,
```

Comment:
removed two lines that were duplicates

Reply:
spacing changes were just a result of `command + option + L` that reformatted the file